sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample4_RecognizeEntities.cs

```diff
@@ -31,7 +31,7 @@ public void RecognizeEntities()
             Console.WriteLine($"Recognized {entities.Count()} entities:");
             foreach (NamedEntity entity in entities)
             {
-                Console.WriteLine($"Text: {entity.Text}, Type: {entity.Type}, SubType: {entity.SubType ?? "N/A"}, Score: {entity.Score}, Offset: {entity.Offset}, Length: {entity.Length}");
+                Console.WriteLine($"Text: {entity.Text}, Type: {entity.Type}, SubType: {entity.SubType.ToString() ?? "N/A"}, Score: {entity.Score}, Offset: {entity.Offset}, Length: {entity.Length}");
```

Comment:
What if SubType had a value of "None" rather than default, and we could just do entity.SubType here instead of checking for null?  If the user wants to specify their own string for None, they can check for None rather than null.

Reply:
In JSON, SubType is not provided in these cases (you can see recorded session). I think maybe we can make this field nullable? Another option is to return "N/A" when "ToString()" is called, but in this case there will be difference between calling `ToString()` and explicit cast.

Reply:
+1 to nullable. It will be more clear